src/QsCompiler/Compiler/Templates/QirDriverCpp.cs

```diff
@@ -170,8 +171,9 @@ unique_ptr<InteropRange> CreateInteropRange(RangeTuple rangeTuple)
     if (arg.CppCliVariableInitialValue() != null) {
         WriteLine($"    {arg.CliValueVariableName()} = {arg.CppCliVariableInitialValue()};");
     }
-    WriteLine($"    app.add_option(\"{arg.CliOptionString()}\", {arg.CliValueVariableName()}, \"{arg.CliDescription()}\")->required()");
+    Write($"    app.add_option(\"{arg.CliOptionString()}\", {arg.CliValueVariableName()}, \"{arg.CliDescription()}\")->required()");
     if (arg.TransformerMapName() != null) {
+    WriteLine("");
         Write($"        ->transform(CLI::CheckedTransformer({arg.TransformerMapName()}, CLI::ignore_case))");
     }
     WriteLine(";");
```

Comment:
Unrelated to your change, but why is this in a WriteLine rather than a Write?

Reply:
it's just to get a newline

Reply:
Looks like it left my comment in the wrong place. I was looking at a WriteLine(";"). But it's not a big deal at all, was just wondering.